llama_stack/core/library_client.py

```diff
@@ -149,36 +148,23 @@ def __init__(
             config_path_or_distro_name, custom_provider_registry, provider_data
```

Comment:
you need to pass `skip_logger_removal` here still

Reply:
fixed 👍🏽